tools/image-builder/build_bond.zsh

```diff
@@ -0,0 +1,82 @@
+#!/bin/zsh
+
+set -eux
+setopt shwordsplit
+
+HOME=$1
+FLAVOR=$2
+BOOST=${3:-}
+
+BUILD_PATH=/root/build/$FLAVOR
+if [ $BOOST != "" ]; then BUILD_PATH=$BUILD_PATH/$BOOST; fi
+
+mkdir -p $HOME $BUILD_PATH
+ln -s /root/.stack $HOME/.stack
+
+cd $BUILD_PATH
+
+case "$FLAVOR" in
+    cpp-*)
+        case "$FLAVOR" in
+            "cpp-core") CPP_CMAKE_ARGS="-DBOND_SKIP_GBC_TESTS=TRUE -DBOND_ENABLE_GRPC=FALSE";;
```

Comment:
~~ I guess the quotes here are unnecessary. Either way is fine with me.

Reply:
Updated